lib/events/dynamoevents/dynamoevents_test.go

```diff
@@ -118,7 +119,7 @@ func (s *DynamoeventsSuite) TestSessionEventsCRUD(c *check.C) {
 	for i := 0; i < dynamoDBLargeQueryRetries; i++ {
 		time.Sleep(s.EventsSuite.QueryDelay)
 
-		history, _, err = s.Log.SearchEvents(s.Clock.Now().Add(-1*time.Hour), s.Clock.Now().Add(time.Hour), apidefaults.Namespace, nil, 0, "")
+		history, _, err = s.Log.SearchEvents(s.Clock.Now().Add(-1*time.Hour), s.Clock.Now().Add(time.Hour), apidefaults.Namespace, nil, 0, types.EventOrderAscending, "")
```

Comment:
Is there coverage for descending order with dynamo and firestore?

Reply:
Yep, added here. https://github.com/gravitational/teleport/blob/joel/event-fetch-order/lib/events/test/suite.go
Existing tests were just filled in with ascending since it was the default previously.